example/misc/7-repo-import/build.sc

```diff
@@ -0,0 +1,33 @@
+import $repo.`file:///tmp/testrepo`
+import mill._, mill.define._, mill.scalalib._
+import coursier.Repository
+
+//object foo extends RootModule {
+def verify(): Command[Unit] = T.command {
+  //  val repos: Seq[Repository] = build.asInstanceOf[ScalaModule].repositoriesTask()
+  os.write(T.dest / "paths.txt", s"${T.workspace} || ${T.dest}")
+  val model = os.read(T.workspace / "out" / "mill-build" / "parseBuildFiles.json")
+  assert(model.contains("""file:///tmp/testrepo""""), "Model was unexpected: " + model)
+}
```

Comment:
The goal of the example tests is to show a semi-realistic workflow that is relevant to the user, while also serving as a test case. 

I think the easiest setup here would be to just manually commit a minimal `file://` repository to the example folder, using the `example/scalamodule/7-unmanaged-jars/lib/nanojson-1.8.jar` file we're already using to keep thing small, but under a different jar name that doesn't exist on maven central. Then we can `import $repo` that local file repo and `import $ivy` our custom jar name and demonstrate usage of `nanojson` inside our `build.sc`

Reply:
Two reasons why I didn't use some other file. I don't know how I can publish some file before the example build.sc is run. And also, I don't know how to refer to a project-local repository as file URL from the `import $repo` statement in the example `build.sc`.

Reply:
Maybe lets move it to the `integration/feature/` folder then? Those are the tests which are meant to assert things but aren't presentable enough for a user to read or include verbatim in the docsite

Reply:
Ok

Reply:
Done